src/store/RelayPendingQueryTracker.js

```diff
@@ -118,12 +118,14 @@ class PendingFetch {
   _errors: Array<?Error>;
 
   constructor(
+    /* eslint-disable comma-dangle */
```

Comment:
Ok! Made a reduced test case and filed an issue eslint/eslint/issues/4634
We could also let the comma be there and let it be found by linter when the bug is fixed.
Btw, used [this eslint plugin diff](https://github.com/eslint/eslint/pull/4419/files#diff-dd33e748952b0c362b1b9fbb73ae1d06R163) for codemodding dangling commas.